src/main.py

```diff
@@ -402,6 +402,7 @@ def metric_page():
     )
 
     fig6.update_layout(yaxis_range=[0, MAE_LIMIT_DEFAULT])
+    st.plotly_chart(fig6, theme="streamlit")
```

Comment:
I added the fig6 MAE/RMSE chart back in for all data models.

Reply:
nice thanks